tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4152,6 +4152,16 @@ def _load_version_file(cls, path) -> str:
         version = load_str.load(path=version_file_path)
         return version
 
+    @classmethod
+    def _load_version_file_old(cls, path) -> str:
```

Comment:
Maybe we could add the logic of trying both old & new filenames to `_load_version_file` instead of introducing a new method `_load_version_file_old`? I feel like the namespace of `Predictor` already contains quite a few methods.

Reply:
fair point, updated